backend/src/main/java/de/jimmy/columbus/model/Route.java

```diff
@@ -0,0 +1,30 @@
+package de.jimmy.columbus.model;
+
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import java.time.Instant;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.List;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Document(collection = "routes")
+public class Route {
+    @Id
+    private String name;
```

Comment:
is a name really unique

Reply:
Thank you for your review ! Gonna fix them asap ! 🙌🏻